backend/development.ini.sample

```diff
@@ -289,7 +289,7 @@ email.notification.created_account.subject = [{website_title}] Someone created a
 email.notification.share_content_to_emitter.subject = [{website_title}] you shared {content_filename} to {nb_receivers} users
 email.notification.share_content_to_receiver.subject = [{website_title}] {emitter_name} shared content {content_filename} with you
```

Comment:
replace `shared content {content_filename} with you` by `shared the file "{content_filename}" with you`

Reply:
done, french translation ?

Reply:
`a partagé le fichier "{content_filename}" avec vous`